### PR TITLE
feat:equal and unequal distribution of expenses with some hook and st…

### PR DESCRIPTION
**Pull request by rishabhparsediya7**
Have added distribution of amounts, percentage is left, will do in next PR.

### CODE REVIEW

File: src/stores/selfStore.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { create } from 'zustand';
+
+type FRIEND_STORE = {
```
Comment (KunalShivhare) [inline src/stores/selfStore.ts:3]: type ka name glt h

File: src/components/members/mebersList.tsx
Comment (KunalShivhare) [inline src/components/members/mebersList.tsx:1]: ye konsi screen h?

Comment (rishabhparsediya7) [reply src/components/members/mebersList.tsx:1]: to show members in the group for the unequal expenses

File: src/stores/selfStore.ts (hunk 1 above)
Comment (rishabhparsediya7) [reply src/stores/selfStore.ts:3]: yes

**Pull request merged**